resources/definitions/hellbot_magna_2_230.def.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": 2,
+    "name": "Hellbot Magna 2 230",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Hellbot Development Team",
+        "manufacturer": "Hellbot",
+        "file_formats": "text/x-gcode",
+	"platform": "Hellbot_Magna_2_230.obj",
+        "platform_texture": "Magna2_230.png",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "hellbot_magna_2_230_extruder_0"
```

Comment:
The Hellbot website claims this printer is dual-extrusion: https://hellbot.xyz/magna-2/especificaciones-magna-2-230/

Why did you include a single-extrusion version as well? Extruders can be disabled in Cura if necessary for a certain print. As far as I could tell, there is no single-extrusion edition of this printer or at least no evidence of it online.

Reply:
Indeed. Most multi-extruder printers can also print single extrusion. Cura allows using only the first extruder, only the second extruder, or both. No need to make a whole new printer for every possible combination of extruders. I'd hate to see an 8-extruder printer making printer definitions for each of the 255 combinations of nozzles that could be used in a print.

I think we'll have to accept it now because previous reviewers didn't seem to have paid attention to it. We'll consider removing the duplicate printers in the future though.